### ISSUE TITLE
Schema / Data Only options for CLI

### ISSUE THREAD

**Issue by mariascott** [gitsql/SQL-Server-Edition#29]
We have received several requests for Schema or Data only options for the CLI.

**Comment by jamesjburton** [comment]
New option added to `opts.json` file named `exportMode` with two valid options of `schema` or `data`.

Example:

```json 
{
        "includeDrop": "false",
        "includeDependencies": "false",
        "includeIndexes" : "false",
        "includeConstraints": "false",
        "includeTriggers": "false",
        "replace":{
                "findMe": "SET QUOTED_IDENTIFIER ON",
                "replaceWith": ""
        },
        "db": "AdventureWorks2012",
        "exportMode": "schema"
} 
```

**Comment by mariascott** [comment]
Released as part of v 1.4.4
